README.md

```diff
@@ -1,9 +1,3 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
 ## Getting Started
```

Comment:
- [ ] @team as previously requested kindly remove this section since you already have getting started section you could choose to add info from this section missing in the other section.